src/services/dashboards/widgets/budget-usage-summary/BudgetUsageSummaryWidget.vue

```diff
@@ -1,6 +1,7 @@
 <template>
     <widget-frame v-bind="widgetFrameProps"
                   :error-mode="false"
+                  :no-height-limit="true"
```

Comment:
```suggestion
                no-height-limit
```
nitpick: The code above shows the same results!

Reply:
praise: 👍 Thx!!